crates/autopilot/src/domain/auction/order.rs

```diff
@@ -12,9 +13,11 @@ pub struct Order {
     pub protocol_fees: Vec<fee::Policy>,
     pub side: Side,
     pub class: Class,
+    pub created_on: Option<DateTime<Utc>>,
     pub valid_to: u32,
     pub receiver: Option<eth::Address>,
     pub owner: eth::Address,
+    pub quote_solver: Option<eth::Address>,
```

Comment:
> Otherwise I'm worried we are scattering information randomly.

That is a fair point, even though the fee policy quote currently contains different information(sell/buy/fee amounts). Especially after your PR with the optional quote https://github.com/cowprotocol/services/pull/2854. 

Although, we need to think about how to make this change seamless. For old auctions, a new DTO will expect a quote as part of the order while it is still in the price improvement fee.

Reply:
`created` and `quote` fields are not optional anymore. As a result, shadow competition might fail on the latest auction right after the release of the feature since an old JSON will be fetched from the DB but all the subsequent JSONs should work fine.